lib/circuit.js

```diff
@@ -168,12 +168,24 @@ class CircuitBreaker extends EventEmitter {
       this[STATUS] = new Status(this.options);
     }
 
-    this[STATE] = CLOSED;
+    if (options.state) {
+      this[ENABLED] = options.state.enabled !== false;
+      this[WARMING_UP] = options.state.warmUp || this[WARMING_UP];
+      this[CLOSED] = options.state.closed || false;
+      this[OPEN] = options.state.open || false;
```

Comment:
> Why do we have both this[OPEN] and this[CLOSED]? 
Well, you wrote it  :)    i would have to look a little deeper to see if both are needed.  maybe that would be in a different PR though

We probably don't need to pass both in.  Since if it isn't closed, then it is Open.   I would opt for only using the closed parameter